### ISSUE TITLE
pytorch code doesn't run on CUDA

### ISSUE THREAD

**Issue by sizhky** [jcjohnson/pytorch-examples#10]
Hi!

I'm trying to run the code in file tensor/two_layer_net_tensor.py which looks similar to this at the beginning
```# Code in file tensor/two_layer_net_tensor.py
import torch

# dtype = torch.FloatTensor
dtype = torch.cuda.FloatTensor # Uncomment this to run on GPU
print('pass1')
# N is batch size; D_in is input dimension;
```

The code works fine when my dtype is ```torch.FloatTensor```, but when I run it on GPU it gets stuck at ```h = x.mm(w1)```. I verified that ```print(torch.cuda.is_available())``` returns true, which means GPU functionality is working fine so I'm at a loss as to what the issue is.

Need help.

**Comment by ArturoDeza** [comment]
Does your system still identify CUDA when you output both:
`$ nvidia-smi`
and
`$ nvcc -V`

**Comment by sizhky** [comment]
There was an issue on our admin side. It's all fine now...
